pkgs/development/libraries/faudio/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, cmake, SDL2}:
+
+#TODO: tests
+
+stdenv.mkDerivation {
+	pname = "faudio";
+  version = "19.09";
+
+  src = fetchFromGitHub {
+    owner = "FNA-XNA";
+    repo = "FAudio";
+    rev = "19.09";
```

```suggestion
    rev = version;
```
with `rec` statements above (see other suggestion) it will works.